lib/data/dev_profile_fields.csv

```diff
@@ -8,7 +8,7 @@ Youtube URL,text_field,https://www.youtube.com/channel/...,,Links
 StackOverflow URL,text_field,https://stackoverflow.com/users/...,,Links
 LinkedIn URL,text_field,https://www.linkedin.com/in/...,,Links
 Behance URL,text_field,https://www.behance.net/...,,Links
-Dribble URL,text_field,https://dribble.com/...,,Links
+Dribbble URL,text_field,https://dribbble.com/...,,Links
```

Comment:
This will be rebased out of here once the other PR is merged, but I needed it to be able to continue.